coralproject/talk
# add ejs templating and serve coral-admin from /admin

## What does this PR do?

coral-admin is now served from `/admin` to make things a bit simpler. the index file of coral-admin is now a very simple template living in the `/views` directory.

## How do I test this PR?

- `npm install`
- `npm run build`
- `npm start`
- visit each of the pages and reload, it should come right back
- wow

@coralproject/tech
